pkg/build/controller/image_change_controller_test.go

```diff
@@ -240,7 +240,7 @@ func TestBuildConfigInstantiatorError(t *testing.T) {
 	if actual, expected := bcInstantiator.newBuild.Parameters.Strategy.DockerStrategy.From.Name, "registry.com/namespace/imagename:newImageID123"; actual != expected {
 		t.Errorf("Image substitutions not properly setup for new build. Expected %s, got %s |", expected, actual)
 	}
-	if bcUpdater.buildcfg != nil {
+	if bcUpdater.buildcfg != nil && bcUpdater.updateCount > 1 {
```

Comment:
The test throws an error here:
https://github.com/csrwng/origin/blob/build_controller_ha/pkg/build/controller/image_change_controller.go#L125
which is after an update happened here:
https://github.com/openshift/origin/pull/2215/files#diff-717854e27d42bef56a972e64d148b529R116
After the update, buildcfg is non-nil